src/hpc/threadapi.c

```diff
@@ -185,11 +185,11 @@ static void WaitThreadSignal(void)
 {
     int id = TLS(threadID);
     if (!UpdateThreadState(id, TSTATE_RUNNING, TSTATE_BLOCKED))
-        HandleInterrupts(1, T_NO_STAT);
+        HandleInterrupts(1, 0);
```

Comment:
Why is it replaced by 0 and not -1 as the value was beforehand?
Does this cause any problems by calling in thread.c:
`SET_BRK_CALL_TO(0);`
instead of 
`SET_BRK_CALL_TO(-1);`?

Sorry, my understanding is not good enough to judge this on my own.

Reply:
This is a very reasonable question, I'm interested too

Reply:
That's of course a reasonable question, thank you for checking closely! I was worried by your question at first, but now I think all is fine: The call to `SET_BRK_CALL_TO` in `InterruptCurrentThread` is never reached in either case. Before this PR, the code does `if (stat == T_NO_STAT) return;`, with this PR it does `if (stat == 0) return;`

Reply:
Ah, right, thank you. And `InterruptCurrentThread` is never called somewhere else with `state`<1?

Reply:
Yes. I just did `git grep InterruptCurrentThread` to verify that `InterruptCurrentThread` is called in exactly one place:
```
$ git grep -n InterruptCurrentThread
src/hpc/thread.c:859:static void InterruptCurrentThread(int locked, Stat stat)
src/hpc/thread.c:904:        InterruptCurrentThread(locked, stat);
```

Reply:
Then this is fine. Thank you.